ios/Framework/Core/Asset/DisplayScreenlet/AssetDisplayBuilder.swift

```diff
@@ -19,16 +19,16 @@ import UIKit
 	open class func createScreenlet(_ frame: CGRect, asset: Asset, themeName: String? = nil)
 			-> BaseScreenlet? {
 
-		if asset.isAnyMimeType(ImageDisplayScreenlet.supportedMimeTypes) {
+		if asset.isAnyMimeType(ImageDisplayScreenlet().supportedMimeTypes) {
 			return ImageDisplayScreenlet(frame: frame, themeName: themeName)
```

Comment:
If you want to use this feature, you have to use a child or inner screenlet such as: ImageDisplayScreenlet, VideoDisplayScreenlet, AudioDisplayScreenlet or PdfDisplayScreenlet.
One of the biggest advantages of AssetDisplayScreenlet is that you don't know the Asset type. So, if you don't know the Asset type, what mime types do you have to override?
This is a good appreciation, by the way 👍